mentorship_task_04_requests.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import argparse
+import os
+
+parser = argparse.ArgumentParser(
+    description='Script that checks all the contributors of a given GitHub project and downloads their avatars into a corresponding folder')
+parser.add_argument('-u', '--user', required=True,
+                    help='user to search for')
+parser.add_argument('-p', '--project', required=True,
+                    help='project to search for')
+
+args = parser.parse_args()
+
+response = requests.get(
+    'https://api.github.com/repos/' + str(args.user) + '/' + str(args.project) + '/contributors'
+)
+os.makedirs('mentorship_task_04_requests_output/' + str(args.user) + '/' + str(args.project), mode=0o777, exist_ok=True)
+
+json_response = response.json()
+contributors_counter = 0
+for contributor in json_response:
+    r = requests.get(contributor['avatar_url'], stream=True)
+    print('Saving', contributor[
+        'login'], 'contributor avatar image')
+    if r.headers['Content-Type'] == 'image/jpeg':
+        f = '.jpg'
+    elif r.headers['Content-Type'] == 'image/png':
+        f = '.png'
+    else:
+        f = ''
+    r = r.raw.read()
+    with open('mentorship_task_04_requests_output/' + str(args.user) + '/' + str(args.project) + '/' + contributor[
```

Comment:
I suggest to use the f-strings to simplify code here - https://realpython.com/python-f-strings/